### DISCUSSION TITLE
마감 기한 정해요

### DISCUSSION THREAD

**Discussion by zangsu** [woowacourse-teams/2024-code-zap#630] [기록해봐요]
모든 태스크에 마감 기한을 정해봅시다.

**Comment by zangsu** [comment]
몰리 : 리뷰 마감 기한만 있는 것이 아니라 Task 마감 기한까지 정해야 한다.

**Comment by zangsu** [comment]
초롱, 제우스 : 리뷰 기한과 Task 마감 기한이 존재하는 것은 현재와 동일하다.

**Comment by zangsu** [comment]
## 결론

현재와 동일하다는 판단 하에 RepositoryTest 에 대한 마감 기한을 정하자.

**Reply by kyum-q** [reply]
### RepositoryTest PR 머지 마감 기한

내일 오후 3시 !
